src/pivot-table/components/PivotTable.tsx

```diff
@@ -137,8 +137,8 @@ export const StickyPivotTable = ({
           style={{
             display: "grid",
             gridTemplateColumns: leftGridWidth // If leftColumnsWidth is 0, this means no data exist for "headers" or "left"
-              ? `${leftGridWidth}px ${rightGridWidth}px`
-              : `${rightGridWidth}px`,
+              ? `${leftGridWidth}px ${rightGridWidth - verticalScrollbarWidth}px`
```

Comment:
Is this not something that should be substracted from in `get-scrollable-areas-dimensions`?